### PR TITLE
normalize/expand descriptions for all commands

### PR DESCRIPTION
**Pull request by pomdtr**

### CODE REVIEW

File: cmd/dev/root.go [hunk 1]
```diff
@@ -49,8 +49,12 @@ var (
 
 func NewCmdDev() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "dev",
-		Short:   "Spin up a local development environment for your Space project",
+		Use:   "dev",
+		Short: "Spin up a local development environment for your Space project",
+		Long: `Spin up a local development environment for your Space project.
+
+The cli will start one process for each of your micros, then expose a single enpoints for your space app.`,
```
Comment (aavshr) [inline cmd/dev/root.go:56]: endpoint~s~

File: cmd/dev/trigger.go [hunk 2]
```diff
@@ -19,8 +19,10 @@ import (
 
 func newCmdDevTrigger() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "trigger <action>",
-		Short:   "Trigger a micro action",
+		Use:   "trigger <action>",
+		Short: "Trigger a micro action",
+		Long: `You can use space trigger to manually trigger action.
```
Comment (aavshr) [inline cmd/dev/trigger.go:24]: to manually trigger _an_ action

File: cmd/dev/root.go (hunk 1 above)
Comment (BetaHuhn) [inline cmd/dev/root.go:56]:
```suggestion
The cli will start one process for each of your micros, then expose a single endpoint for your Space app.`,
```

File: cmd/exec.go [hunk 3]
```diff
@@ -13,8 +13,11 @@ import (
 func newCmdExec() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "exec",
-		Short: "executes a command in space context",
-		Args:  cobra.MinimumNArgs(1),
+		Short: "Run a command in the context of your project",
+		Long: `Run a command in the context of your project.
+
+The project key will be automatically injected into command env.`,
```
Comment (aavshr) [inline cmd/exec.go:19]: into _the_ command _environment_

File: cmd/dev/trigger.go (hunk 2 above)
Comment (BetaHuhn) [inline cmd/dev/trigger.go:24]:
```suggestion
		Long: `You can use space trigger to manually trigger a action.
```

File: cmd/link.go [hunk 4]
```diff
@@ -18,7 +18,7 @@ import (
 func newCmdLink() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "link [flags]",
-		Short: "link code to project",
+		Short: "Link a directory with a existing project",
```
Comment (aavshr) [inline cmd/link.go:21]: with _an_ existing project

File: cmd/exec.go [hunk 5]
```diff
@@ -13,8 +13,11 @@ import (
...
+		Args: cobra.MinimumNArgs(1),
```
Comment (BetaHuhn) [inline cmd/exec.go:20]:
```suggestion
		Long: `Run a command in the context of your project.

The data key will be automatically injected into the command's environment.`,
		Args: cobra.MinimumNArgs(1),
```

File: cmd/release.go [hunk 6]
```diff
@@ -24,7 +24,7 @@ const (
 func newCmdRelease() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "release [flags]",
-		Short:   "create release for a project",
+		Short:   "Create new releases out of revisions",
```
Comment (BetaHuhn) [inline cmd/release.go:27]:
```suggestion
		Short:   "Create a new release from a revision",
```

**Review by aavshr — COMMENTED** [review]
thanks @pomdtr just a few nitpicks

**Comment by pomdtr** [comment]
Wow so much spelling mistakes 😅

**Review by aavshr — APPROVED** [review]

**Pull request merged**